guide/daily-spending-wallet/requesting.md

```diff
@@ -34,439 +34,454 @@ Illustration source
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 </div>
 
-## Single-use payment requests
+## Entering an amount
 
-The most common way bitcoin is requested is in the form of single-use payment requests. Lightning payments use [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoices. On-chain uses [BIP21](https://bips.xyz/21) and [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}). A BOLT11 invoice can be [embedded into a BIP21 URI](https://github.com/sbddesign/bip21-site) to form a single payment request so senders can use what is most convenient for them and requesters do not have to be worried about requesting from on-chain or lightning.
+Once the user is on the requesting entry point, the next thing to do is add an amount to be requested. Users should be able easily toggle between entering amounts in sats, bitcoin, or the users local currency by single tapping the amount field. More on entering amounts on our [Units & Symbols page]({{ "/guide/designing-products/units-and-symbols/" | relative_url }}).
 
-Single-use payment requests are convenient for one-time or in-person payments. Generating them should be fast and convenient from the users home screen. Wallet balances should not be shown when generating single-use payment requests. This is due to the users funds being visible to nearby third-parties, which negatively impacts their privacy and puts the user at risk of theft.
+Users can also skip entering an amount, as there are scenarios where the sender is deciding the amount to send. No amount requests also limit both parties exposure to price volatility as the bitcoin price in fiat terms could change between requesting and the payment being sent.
 
 <div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/home.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/home@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/HomeEuro.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/HomeEuro@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/HomeEuro@2x.png"
    layout = "shadow"
-   caption = "Hiding the balance and making it easy to generate a single-use payment request from the home page is recommended."
-   alt-text = "Screen showing home screen number pad entry."
+   caption = "Let users request in fiat currency."
+   alt-text = "Screen showing home screen with a fiat euro input selected."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/HomeBTC.png"
```

Comment:
The "delete character" icon on the left screen is different than the one on the right screen.

Reply:
Will update, missed that